zproject/computed_settings.py

```diff
@@ -1120,6 +1120,8 @@ def zulip_path(path: str) -> str:
 
 SOCIAL_AUTH_GITHUB_SECRET = get_secret("social_auth_github_secret")
 SOCIAL_AUTH_GITLAB_SECRET = get_secret("social_auth_gitlab_secret")
+SOCIAL_AUTH_AZUREAD_OAUTH2_SECRET = get_secret("social_auth_azuread_oauth2_secret")
```

Comment:
As mentioned in the commits, message here this got renamed from `azure_oauth2_secret` to `social_auth_azuread_oauth2_secret` which will break existing configurations. I suppose we have 2 ways to go:
1. We could add a line in `computed_settings` to keep fetching from  `azure_oauth2_secret` for back-compatibility
2. It's possible that there aren't any (or many) deployments using it since it wasn't particularly advertized, docs were outdated and i don't think we've had folks mentioning it in czo :thinking:

Reply:
I think we can just document this in "Upgrade notes".